oracle/price-feeder/oracle/oracle.go

```diff
@@ -116,19 +119,24 @@ func (o *Oracle) Start(ctx context.Context) error {
 		default:
 			o.logger.Debug().Msg("starting oracle tick")
 
-			startTime := time.Now()
+			// Wait for next block height to be available in the channel
+			nextBlockHeight := <-o.oracleClient.BlockHeightEvents
```

Comment:
nit, shouldnt this be `currBlockHeight` not next block height?

Reply:
will fix